meta-mender-core/recipes-mender/mender-client/mender-client-cpp_git.bb

```diff
@@ -0,0 +1,14 @@
+require mender-client-cpp.inc
```

Comment:
Question is whether or not we should replace this with the `golang` mender-client_git recipe at this point in time.